tests/services/test_feed_manager.py

```
@@ -0,0 +1,84 @@
+        new_feed = Feed(title="New Feed", url="http://newfeed.com", genre="News")
+        feed_manager.feeds[new_feed.hash] = new_feed
+        assert len(feed_manager.feeds) == 2
+
+        del feed_manager.feeds[new_feed.hash]
+        assert len(feed_manager.feeds) == 1
+
+        feed_manager.feeds[new_feed.hash] = new_feed
+        feed_manager.feeds[new_feed.hash].title = "Updated Feed"
+        assert feed_manager.feeds[new_feed.hash].title == "Updated Feed"
+
+def test_move_to_deleted(feed_manager, valid_opml_content):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_move_to_deleted`

Add return type annotation: `None`

(ANN201)